scripts/setup.sh

```diff
@@ -572,12 +567,10 @@ if [[ "$UPGRADE_EXTENSIONS" != "Y" && -n "$EXTENSIONS_FROM_CLUSTER" ]]; then
   done
 fi
 
-# Add '/default' to service name when featureSet is missing
```

Comment:
Why did you remove it? Shouldn't we change `default` to `default_metrics` here?
Line 586 should also be change anyway.

Reply:
We don't want to add any featureSet by default. FeatureSet for service should be defined explicitly.

Reply:
We don't want to add any featureSet by default. FeatureSet for service should be defined explicitly.